File: 706B - Interesting drink.py
#110157609	Mar/17/2021 14:46UTC+5.5	Shan_XD	706B - Interesting drink	PyPy 3	Accepted	1372 ms	10500 KB

import bisect
n=int(input())
x=list(map(int,input().split()))
x.sort()
q=int(input())
for i in range(q):
    m=int(input())
    print(bisect.bisect_right(x,m))

File: 706B - Interesting drink.cpp
//110157726	Mar/17/2021 14:47UTC+5.5	Shan_XD	706B - Interesting drink	GNU C++17 (64)	Accepted	576 ms	400 KB

#include <bits/stdc++.h>
 
using namespace std;
 
int main()
{
    int n,q,i,j,x;
    cin>>n;
    int a[n+10];
    for(i=0;i<n;i++){
        cin>>a[i];
    }
    cin>>q;
    sort(a,a+n);
    for(i=0;i<q;i++){
       cin>>x;
       int ans=upper_bound(a,a+n,x)-a;
       cout<<ans<<endl;
    }
    return 0;
}
